src/Export.php

```diff
@@ -49,8 +68,14 @@ private static function getHeaders(Collection $items)
     {
         $headers = new Collection;
 
+        $headers->push('id');
+
```

Comment:
Wasn't the ID already exported? If not, how do we insert ID data in the entires?

Reply:
it wasn't, and the ID is generated by statamic, we don't need to add it manually in the entries